package.json

```diff
@@ -21,7 +21,7 @@
     "@bcrs-shared-components/corp-type-module": "1.0.11",
     "@bcrs-shared-components/court-order-poa": "2.1.4",
     "@bcrs-shared-components/document-delivery": "1.2.1",
-    "@bcrs-shared-components/enums": "1.0.38",
+    "@bcrs-shared-components/enums": "1.0.39",
```

Comment:
Check the other shared components that were updated due to their dependency on the shared enums. If their version only went up by 1 then please update it here. If it's more than 1 then that means other changes happened and testing is needed -- not in scope -- leave them at their original version.